src/plugins/intel_gpu/src/kernel_selector/kernels/gather/gather_kernel_ref.cpp

```diff
@@ -186,9 +186,21 @@ static std::string GetIndicesIdxOrder(const gather_params& params, size_t axis,
 
     size_t indices_dims_num = GetNonEmptyDimsNumber(params.inputs[1]);
 
-    // Shift indices of Gather indices input related to output dims
```

Comment:
[random spot] Please add functional test case that covers fixed issue here: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/tests/functional/shared_tests_instances/single_layer_tests/gather.cpp

Reply:
Added the earlier failure tensorflow_tests that uncover the bug.  Should I still need to add more test in gather.cpp?